Podfile

```diff
@@ -11,3 +11,11 @@ target 'AppBidding-GoogleAdManager' do
   pod 'Google-Mobile-Ads-SDK', '~> 7.60'
   pod 'CriteoPublisherSdk', '~> 3.6'
 end
+
+project 'AppBidding-MoPub/AppBidding-MoPub.xcodeproj'
+target 'AppBidding-MoPub' do
+  project 'AppBidding-MoPub/AppBidding-MoPub.xcodeproj'
+  use_frameworks!
+
+  pod 'mopub-ios-sdk'
```

Comment:
We may want to specify MoPub SDK version to avoid breaking changes on future MoPub SDK.
I suggest we use `'~> 5.13'`